qt/scientific_interfaces/Muon/ALCDataLoadingPresenter.cpp

```diff
@@ -326,7 +323,18 @@ void ALCDataLoadingPresenter::updateAvailableInfo() {
 
   // sort alphabetically
   std::sort(logs.begin(), logs.end(), [](std::string &logA, std::string &logB) {
-    return std::tolower(logA[0]) < std::tolower(logB[0]);
```

Comment:
Could I ask why  simply `std::sort(logs.begin() logs.end())` isn't sufficient?

`operator<` is defined for [std::string](https://en.cppreference.com/w/cpp/string/basic_string/operator_cmp) so by default `sort` should use that.

Reply:
Because some of the logs are all capitalised and some are not, which means the end result is not fully alphabetised using the standard sort

Reply:
Yes I see, I somehow didn't see the `tolower`. Could we add a comment about that?

There is a nice example of this on [stackoverflow](https://stackoverflow.com/a/33380725) using just STL algorithms without the need for our own `for` loop. They show an example in an [`online IDE`](http://ideone.com/BVgyD2).